egs/librispeech/ASR/zipformer/train.py

```diff
@@ -1234,14 +1296,14 @@ def remove_short_and_long_utt(c: Cut):
     valid_cuts += librispeech.dev_other_cuts()
     valid_dl = librispeech.valid_dataloaders(valid_cuts)
 
-    if not params.print_diagnostics:
-        scan_pessimistic_batches_for_oom(
-            model=model,
-            train_dl=train_dl,
-            optimizer=optimizer,
-            sp=sp,
-            params=params,
-        )
+    # if not params.print_diagnostics:
```

Comment:
is there a particular reason this was commented?

Reply:
It was just for speed during my exp. Will make sure to un-comment them when I merge the PR.